docs/jax.profiler.rst

```diff
@@ -1,6 +1,27 @@
+.. currentmodule:: jax.profiler
+
 jax.profiler module
 ===================
 
 .. automodule:: jax.profiler
-    :members:
-    :show-inheritance:
\ No newline at end of file
+
```

Comment:
Consider linking to the Profiling and Heap Profling pages here (or under the respective titles).